dev-app.js

```diff
@@ -62,7 +63,7 @@ app.get("/", (req, res) => {
       if (lngCookie) {
         lngCookieValue = getCookieValue(lngCookie.split(";"), "lng");
         res.cookie("lng", lngCookieValue, {
-          maxAge: new Date(new Date().getTime() + 60 * 60000),
```

Comment:
maxAge rather than expires, currently falling over for me as expires takes a date